mappings.py

```diff
@@ -140,50 +209,91 @@ def boolean(data_values):
 
 
 def integer(data_values):
+    """Convert a value to an integer.
+
+    Args:
+        data_values: a values dict with value to be converted to an int
+    Returns:
+        an integer version of the input value
+    Raises:
+        ValueError if int() cannot convert the input
+    """
     cell = single_val(data_values)
     if cell is None or cell.lower() == "nan":
         return None
     try:
         return int(cell)
-    except:
+    except ValueError as e:
+        _warn(e)
         return None
 
 
 def float(data_values):
-    cell = single_val(data_values)
-    if cell is None or cell.lower() == "nan":
-        return None
-    try:
-        return float(cell)
-    except:
-        return None
+    """Convert a value to a float.
 
+    Args:
+        data_values: A values dict
 
-def double(data_values):
+    Returns:
+        A values dict with a string or integer converted to a float or None if null value
+
+    Raises:
+        ValueError by float() if it cannot convert to float.
+    """
     cell = single_val(data_values)
     if cell is None or cell.lower() == "nan":
         return None
     try:
         return float(cell)
-    except:
+    except ValueError as e:
+        _warn(e)
         return None
 
 
-# Placeholder function to make a fake ontology entry
+# This seems like a duplicate of the method above, ok to delete?
```

Comment:
Is it okay to delete this method?

Reply:
Yeah, I couldn't decide which was a more-common word. Go ahead and delete.

Reply:
ok will do